pkgs/servers/sql/mariadb/connector-c/default.nix

```diff
@@ -52,6 +52,7 @@ stdenv.mkDerivation {
     ln -sv mariadb $dev/include/mysql
     ln -sv mariadb_version.h $dev/include/mariadb/mysql_version.h
     ln -sv libmariadb.pc $dev/lib/pkgconfig/mysqlclient.pc
```

Comment:
If we symlink the files like this doesn't that make src a depedency of the final package?

Reply:
> If we symlink the files like this doesn't that make src a depedency of the final package?

we're not symlinking them from the src, instead we're making relative symlinks 
```
ls -l mysql
lrwxrwxrwx root root 7 B Thu Jan  1 02:00:01 1970  mysql@ ⇒ mariadb
```